pyproject.toml

```diff
@@ -0,0 +1,157 @@
+# SPDX-License-Identifier: BSD-2-Clause
+[project]
+name = 'torii-boards'
+description = 'Supplementary FPGA Board Files for Torii'
+authors = [
+	{ name = 'Aki Van Ness', email = 'aki@lethalbit.net' },
+	{ name = 'Rachel Mant', email = 'git@dragonmux.network' },
+	{ name = 'Torii Contributors' },
+]
+dynamic = ['version']
+license = 'BSD-2-Clause'
+license-files = ['LICENSE', 'LICENSE.docs']
+readme = 'README.md'
+requires-python = '>=3.11'
+dependencies = [
+	'torii>=0.8.0,<1.0',
+]
+keywords = [
+	'Torii',
+	'HDL',
+	'FPGA',
+]
+classifiers = [
+	'Development Status :: 4 - Beta',
+
+	'Intended Audience :: Developers',
+	'Intended Audience :: Education',
+	'Intended Audience :: Information Technology',
+	'Intended Audience :: Science/Research',
+
+	'Operating System :: MacOS :: MacOS X',
+	'Operating System :: Microsoft :: Windows',
+	'Operating System :: POSIX :: Linux',
+
+	'Programming Language :: Python :: 3.11',
+	'Programming Language :: Python :: 3.12',
+	'Programming Language :: Python :: 3.13',
+	'Programming Language :: Python :: 3.14',
+
+	'Programming Language :: Python :: Implementation :: CPython',
+	'Programming Language :: Python :: Implementation :: PyPy',
+
+	'Topic :: Scientific/Engineering',
+	'Topic :: Scientific/Engineering :: Electronic Design Automation (EDA)',
+	'Topic :: Software Development',
+	'Topic :: Software Development :: Embedded Systems',
+	'Topic :: Software Development :: Libraries',
+
+	'Typing :: Typed',
+]
+
+[project.optional-dependencies]
+dev = ['nox']
+
+[project.urls]
+source = 'https://github.com/shrine-maiden-heavy-industries/torii-boards'
+issues = 'https://github.com/shrine-maiden-heavy-industries/torii-boards/issues'
+documentation = 'https://torii-boards.shmdn.link'
+changelog = 'https://torii-boards.shmdn.link/changelog.html'
+
+[build-system]
+requires = ['setuptools>=66', 'setuptools-scm>=8']
```

Comment:
Do we actually care about the minimum setuptools version here? we understand the constraint on Torii itself, but this feels like a slight over-constraint here (it's transitively implied by Torii's one).

Reply:
It's not, as that's a build-time dep, and we do need to constrain it for setuptools-scm to work.